cron/update_lunch.py

```diff
@@ -1,18 +1,28 @@
 import datetime
 import logging
+import os
+import os.path
 
+import anvil
 import requests
+from google.cloud import ndb
+
+# os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "service_account.json"

```

Comment:
I think you can remove this stuff, and we'll just set up the credentials as we do in the other cron scripts (see my comment below).